### PR TITLE
Resolution of Issue#3010

### PR DESCRIPTION
**Pull request by j0rt3g4**
Resolution of Issue #3010 using an important note.

### CODE REVIEW

**Comment by illfated** [comment]
(Ref. PR #3190, replaced by this PR.)

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md [hunk 1]
```diff
@@ -34,5 +34,8 @@ To trigger a remote Windows Autopilot Reset via Intune, follow these steps:
 >[!NOTE]
 >The Autopilot Reset option will not be enabled in Microsoft Intune for devices not running Windows 10 build 17672 or higher.
 
+>[!IMPORTANT]
+>To use the Autopilot Reset option your device must be enrolled in a Autopilot Device.
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md]: To use the Autopilot Reset **option,** your device must be enrolled in **an** Autopilot **device**.

Kindly do changes in bold (insert comma, change a to an, and change upper case D to lower case d in device).

**Comment by mypil** [comment]
@j0rt3g4 - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions he made. Thank you.

**Comment by mypil** [comment]
@j0rt3g4 - I would like to follow up on this. Have you already merged the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions he made?

File: windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md (hunk 1 above)
Comment (illfated) [reply windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md]: Practical way of showing it: use the "Insert a suggestion `<Ctrl+G>`" feature:
```suggestion
>To use the Autopilot Reset option, your device must be enrolled in an Autopilot device.
```

**Comment by j0rt3g4** [comment]
@mypil The updates have been committed.

**Comment by mypil** [comment]
@greg-lindsay - Can you please assist to check, approve and merge the content for this PR based on issue #3010?

Thank you.

File: windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md (hunk 1 above)
Comment (nenonix) [reply windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md]: @illfated Thank you, will investigate this as a way to improve our process.

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR? Thank you.

**Comment by AndreaBarr** [comment]
@mypil I have contacted Greg via email to review this PR since it adds a note that I can't verify myself.

**Comment by j0rt3g4** [comment]
Added  comments @mypil

File: windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md [hunk 2]
```diff
@@ -34,5 +34,8 @@ To trigger a remote Windows Autopilot Reset via Intune, follow these steps:
 >[!NOTE]
 >The Autopilot Reset option will not be enabled in Microsoft Intune for devices not running Windows 10 build 17672 or higher.
 
+>[!IMPORTANT]
+>To use the Autopilot Reset (preview) option, your device **must be** reset using Autopilot *(either using Fresh Reset or manually sysprep the device)*, in any other case the feature will stay **grayed out**.
```
Comment (illfated) [inline windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md]: I must admit that I prefer @Neckross' original phrasing:
```suggestion
>The feature for Autopilot Reset (preview) will stay grayed out, **unless** you reset the device using Autopilot (either using Fresh Reset or manually sysprep the device).
```
My suggestion is not the blueprint or final answer. The MS Doc team members (or associates) might have a better suggestion for this added note.

Comment (mypil) [reply windows/deployment/windows-autopilot/windows-autopilot-reset-remote.md]: @j0rt3g4 - Kindly check on the suggestion from @illfated and make the necessary changes. Thank you.

**Comment by mypil** [comment]
@greg-lindsay - We would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #3010?

Thank you.

**Pull request merged**